engine/Dockerfile

```diff
@@ -1,4 +1,10 @@
 FROM python:3.11.3-slim-buster AS base
+
+# Create a group and user to run an app
+ENV APP_USER=appuser
+RUN groupadd --system --gid 1000 ${APP_USER} && \
```

Comment:
can we keep gid=2000 to make sure it is backwards-compatible?

Reply:
Yes, we can change to `gid=2000`. 

I think it's a common practice to set the same `uid` and `gid` for a custom user. I suppose that nothing would break, as process that starts application have the same user permissions. Something can be broken, if another process with some other uid and in the same group wants to use files with group permissions. But I suppose it's not applicable to dockerized application.

I'll change to `gid=2000` if you prefer that.

Reply:
Hi @iskhakov @joeyorlando ! Change to `gid=2000` or not ?

Reply:
I agree w/ @iskhakov , lets keep it to 2000 just to be safe